### PR TITLE
OCPBUGS-34135#adding required permission for creating compute resources

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions: 
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-34135

Link to docs preview:
- https://77148--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#minimum-required-permissions-ipi-azure_installing-azure-account
- https://77148--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra.html#minimum-required-permissions-upi-azure_installing-azure-user-infra
- https://77148--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-restricted-networks-azure-user-provisioned.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/minimum-required-permissions-ipi-azure.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ The following permissions are required for creating an {product-title} cluster o
 .Required permissions for creating compute resources
 [%collapsible]
 ====
-* `Microsoft.Compute/availabilitySets/write`
```
Comment (sunzhaohua2) [inline modules/minimum-required-permissions-ipi-azure.adoc:32]: We still need `Microsoft.Compute/availabilitySets/write`

**Comment by sunzhaohua2** [comment]
Hi @brendan-daly-red-hat  sorry I missed some info in bug. For the 3 doc "Required permissions for creating compute resources" part we should have below 3 permissions:

- Microsoft.Compute/availabilitySets/delete
- Microsoft.Compute/availabilitySets/read
- Microsoft.Compute/availabilitySets/write

https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-account.html 
https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-user-infra.html
https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-restricted-networks-azure-user-provisioned.html 

and remove `Microsoft.Compute/availabilitySets/write` from "Optional permissions for creating compute resources" part in below doc

https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-user-infra.html
https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-restricted-networks-azure-user-provisioned.html

**Comment by brendan-daly-red-hat** [comment]
The 'Microsoft.Compute/availabilitySets/write' update refers to a request that was originally in the Jira issue but then removed. (I deleted but then re-instated 'Microsoft.Compute/availabilitySets/write'.)

**Comment by skrthomas** [comment]
@brendan-daly-red-hat if this no longer needs peer review, can you please remove the peer-review-needed label?

**Comment by GroceryBoyJr** [comment]
@brendan-daly-red-hat yes please remove it. I almost started another.

**Comment by brendan-daly-red-hat** [comment]
> @brendan-daly-red-hat if this no longer needs peer review, can you please remove the peer-review-needed label?

Hi Sara. Apologies, this PR does need a peer review (I inadvertently asked for a merge review).

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**